boot/pod/src/boot/pod.clj

```diff
@@ -339,37 +342,42 @@
    [#".*"                   first-wins-merger]])
 
 (defn merge-duplicate-jar-entry
-  [mergers curr-file new-path new-url & {:keys [merged-url]}]
+  [mergers curr-file new-path new-stream & {:keys [merged-url]}]
```

Comment:
Does this break [the JAR entry &rarr; fileset uber merging code](https://github.com/boot-clj/boot/blob/09eadb04a55e3602daf553d72626f57860f49cb8/boot/pod/src/boot/tmpdir.clj#L84)?